src/Domain/AuthorisationService.cs

```diff
@@ -1,31 +1,31 @@
-﻿namespace Linn.Products.Domain
+namespace Linn.Products.Domain
 {
     using System.Collections.Generic;
     using System.Linq;
 
     public class AuthorisationService : IAuthorisationService
     {
-        public bool HasPermissionFor(AuthorisedAction action, IEnumerable<string> privileges)
+        public bool HasPermissionFor(string action, IEnumerable<string> privileges)
         {
             switch (action)
             {
                 case AuthorisedAction.VatAdmin:
-                    return this.CanEditOrCreateVatCodes(privileges);
-                    break;
+                    return this.CanEditOrCreate(AuthorisedAction.VatAdmin, privileges);
+                case AuthorisedAction.TariffAdmin:
+                    return this.CanEditOrCreate(AuthorisedAction.TariffAdmin, privileges);
                 default:
                     return false;
-                    break;
             }
         }
 
-        private bool CanEditOrCreateVatCodes(IEnumerable<string> privileges)
+        private bool CanEditOrCreate(string action, IEnumerable<string> privileges)
```

Comment:
I don't think you need this anymore. Could go straight to the satisfies method?

Reply:
👍